bokeh/server/tornado.py

```diff
@@ -120,6 +145,7 @@ def __init__(self, applications, prefix, hosts,
             raise ValueError("stats_log_frequency_milliseconds must be > 0")
 
         self._hosts = set(hosts)
+        self._hosts.add('127.*.*.*')
```

Comment:
My understanding is that all `127.*.*.*` addresses connect directly to loopback.  It would be challenging for an attacker to spoof itself as one of these trusted addresses.  Telling the Bokeh server that it is on `127.*.*.*` seems like it provides no new information.

My web security experience is pretty light though.  I may not fully appreciate the situation.